deepspeed/autotuning/scheduler.py

```diff
@@ -5,7 +5,11 @@
 
 import copy
 
-from numpy import BUFSIZE
+import numpy
+if numpy.__version__ < '2.0.0':
```

Comment:
@Yejing-Lai - we are working to determine if we need BUFSIZE from this file or from deepspeed/autotuning/constants.py (which is used in deepspeed/autotuning/autotuner.py), if we do need this from np, we will merge this.